src/FSharp.Core/FSharp.Core.php

```diff
@@ -85,10 +85,10 @@ static function int32ToString($x)
     }
 }
 interface Union {
-    public function get_Case();
+    public static function get_Case();
```

Comment:
Why is this static ? The goal is to get the case of current instance ?
Does static have a different meaning on PHP interfaces ?

Reply:
Static allows us to call the function on the class level (as well as the instance). And given the function on the actual Union classes are static, this one must be declared as such as well.

The main reason to do this is to get from the PHP class name to the F# "tag" without creating any instances.

Reply:
In the end, for `get_Case` it's not needed anymore, only for `get_FSharpCase` probably 👍 (but it shouldn't hurt either given static methods can be called on the instance as well)